source/RuuviTagSensor.mc

```diff
@@ -61,10 +62,14 @@ class RuuviTagSensorView extends WatchUi.View {
     ]);
   }
 
+  private function getSensorAlias(macAddress) {

```

Comment:
Could be called "getSensorDisplayName()", since it's not returning the alias in all cases.